auto_backup/view/db_backup_view.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_backup_conf_form" model="ir.ui.view">
+        <field name="model">db.backup</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button
+                        name="action_backup"
+                        type="object"
+                        string="Execute backup"
+                        class="oe_highlight"
+                    />
+                </header>
+                <div class="oe_title">
+                    <h1>
+                        <field name="name" />
+                    </h1>
+                </div>
+                <group string="Basic backup configuration">
+                    <field name="folder" />
+                    <field name="days_to_keep" />
+                    <field name="method" />
+                    <field name="backup_format" />
+                </group>
+                <div invisible="method != 'sftp'">
+                    <div class="bg-warning">
+                        <h3>Warning:</h3>
+                        Use SFTP with caution! This writes files to external servers under the path you specify.
+                    </div>
+                    <group string="SFTP Settings">
+                        <field name="sftp_host" placeholder="sftp.example.com" />
+                        <field name="sftp_port" />
+                        <field name="sftp_user" placeholder="john" />
+                        <field name="sftp_password" />
+                        <field
+                            name="sftp_private_key"
+                            placeholder="/home/odoo/.ssh/id_rsa"
+                        />
+                        <button
+                            name="action_sftp_test_connection"
+                            type="object"
+                            string="Test SFTP Connection"
+                            icon="fa-television"
+                        />
+                    </group>
+                </div>
+                <separator string="Help" colspan="2" />
+                <div>
+                    Automatic backups of the database can be scheduled as follows:
+                    <ol>
+                    <li
+                    >Go to Settings / Technical / Automation / Scheduled Actions.</li>
+                    <li>Search the action named 'Backup scheduler'.</li>
+                    <li
+                    >Set the scheduler to active and fill in how often you want backups generated.</li>
+                </ol>
+                </div>
+            </form>
```

Comment:
add:
</sheet>
 <div class="oe_chatter">
                        <field name="message_follower_ids" groups="base.group_user"/>
                        <field name="message_ids"/>
                    </div>

Reply:
thanks, added!

Should be ok now